principles/fp-020-responsiveness.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: responsiveness
+id: fp-020-responsiveness
+title: Responsiveness (principle 20)
+---
+
+NOTE
+-------
+The wording of this principle is still in progress.
+
+Summary
+-------
+Ontology developers MUST offer channels for community participation and be responsive to requests. 
+
+[This check is automatically validated.](checks/fp_020)
+
+Purpose
+-------
+Ontology development benefits from community input, which is strongly encouraged by the OBO Foundry. Accordingly, "responsiveness" is a key quality of our general collaborative spirit. This principle is intended to guide the processes to ensure community responsiveness. 
+
+Recommendation
+------- 
+Ontology developers SHOULD aim for a good level of responsiveness to community requests and suggestions.
+
+Expectations of responsiveness:
+1. Issues are contributions - and should be met by a thankful attitude. When receiving a ticket on your issue tracker, the first thing to do is thank the contributor, even if a ticket cannot be addressed right away. 
+1. If aa ticket cannot be addressed right away, eplain when you plan to look at the ticket in more detail. This should happen soon after a ticket is issued - ideally in the first 2-5 working days after the issue entered your tracker. 
+1. Please do not close tickets without responding. If a ticket is out of scope for a repository, recommend moving it to a different repository.
+1. It is recommended that someone from each ontology is responsible for triaging issues and requests.
+
+Implementation
+-------
+A discussion mailing list and issue tracker are required to obtain an OBO Foundry PURL. The OBO Foundry offers the following recommendations on the responsiveness in issue trackers (GitHub is recommended).
+
+1. Establish an issue tracker (GitHub is recommended).
+2. Establish a mailing list (For example, google groups).
+3. The issue tracker and mailing list MUST be monitored and responded to in a reasonable amount of time (ideally 2-5 working days).
```

Comment:
so MUST be responded to in a reasonable amount of time, but that amount of time is only suggested, not required?